bedrock/mozorg/templates/mozorg/update-browser.html

```diff
@@ -0,0 +1,80 @@
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+ {% extends "base-protocol.html" %}
+
+ {% from "macros-protocol.html" import call_out_compact with context %}
+
+ {% block page_title %}{{ _('Update your browser to fast, safe and secure Firefox.') }}{% endblock %}
+
+ {% block page_desc %}
+   {{ _('Get the fastest Firefox yet for protection against hacks, powerful privacy extensions and exciting new features. Update to a browser that fights for you.') }}
+ {% endblock %}
+
+ {% block page_css %}
+   {{ css_bundle('update-browser') }}
+ {% endblock %}
+
+ {% block content %}
+   <div class="mzp-l-content">
+   <article class="mzp-c-article">
+     <h1 class="mzp-c-article-title">{{ self.page_title() }}</h1>
+     <p class="mzp-c-article-intro">{{ _('One of the most important things you can do to have a safe, fast and secure online browsing experience is to make sure your browser is up to date. Update your browser like you would update your apps. No matter which browser you use, make sure you’re using the latest version.') }}</p>
+
+     <img src="{{ static('img/mozorg/technology/fast.png') }}" alt="" class="c-article-image">
+
+     <h2 class="c-section-title">{{ _('Stay safe, browse safe') }}</h2>
+     <p>
+     {% trans security="https://hacks.mozilla.org/2018/03/shipping-a-security-update-of-firefox-in-less-than-a-day/"%}
+        Up-to-date browsers protect you from viruses, security breaches and hacks. Older versions of browsers may be vulnerable to attacks and security holes. Firefox engineers have been known to ship a security update <a href="{{ security }}">within a day</a> of learning of a vulnerability.
+     {% endtrans %}
+     </p>
+
+     <h2 class="c-section-title">{{ _('The fastest Firefox yet') }}</h2>
+     <p>
+     {% trans %}
+        We work tirelessly to make sure Firefox is the fastest it can be, while making sure it doesn’t hog your memory or system resources. With each version we make improvements to the code that makes Firefox quick and nimble as you browse.
+     {% endtrans %}
+     </p>
+
+     <h2 class="c-section-title">{{ _('The best privacy') }}</h2>
+     <p>
+     {% trans privacy="https://blog.mozilla.org/firefox/make-your-firefox-browser-a-privacy-superpower-with-these-extensions/"%}
+        We are passionate about user privacy. With each new release we give you more ways to control who sees and accesses your personal browsing data. Tracking protection, private browsing and <a href="{{ privacy }}">powerful privacy extensions</a> all work together to make sure your private browsing information stays yours.
+     {% endtrans %}
+     </p>
+
+     <h2 class="c-section-title">{{ _('New features') }}</h2>
+     <p>
+     {% trans accessibility="https://support.mozilla.org/en-US/kb/accessibility-features-firefox-make-firefox-and-we", extensions="https://blog.mozilla.org/firefox/its-a-new-firefox-multi-tasking-extension-side-view/" %}
```

Comment:
These links could have the `/en-US/` locale removed, so they display in the visitors preferred language. Not a blocker though, seeing as this page is en-US only.